app/views/drafts/index.html.erb

```diff
@@ -14,7 +14,7 @@
   Draft records (<span id="js-trainee-count"><%= total_trainees_count %></span><span class="govuk-visually-hidden"> <%= "record".pluralize(total_trainees_count) %></span>)
 </h1>
 
-<% unless current_user.system_admin? %>
+<% if current_user.provider? %>
```

Comment:
We could but it will enable system admins to create trainees (the button was hidden for them previously), i'm not sure if that's a concern, but happy to make the change.